pkg/controllers/dynakube/connectioninfo/reconciler.go

```diff
@@ -36,19 +36,32 @@ func NewReconciler(clt client.Client, apiReader client.Reader, scheme *runtime.S
 	}
 }
 
+func (r Reconciler) updateDynakubeStatus(ctx context.Context) {
+	r.dynakube.Status.UpdatedTimestamp = metav1.Now()
+	err := r.client.Status().Update(ctx, r.dynakube)
+	if err != nil {
+		log.Error(err, "could not update dynakube status", "name", r.dynakube.Name)
+	}
+}
```

Comment:
there is already a function like this ([its also private](https://github.com/Dynatrace/dynatrace-operator/blob/141ae33217b78bf6bdcd054a57cb94777f96313c/pkg/controllers/dynakube/dynakube_controller.go#L451-L451)), could we somehow reuse it 🤔 ?

also, shouldn't we return the error?